systems/subscriber/registry/pkg/db/model.go

```diff
@@ -11,18 +11,22 @@ package db
 import (
 	"time"
 
+	"github.com/ukama/ukama/systems/common/ukama"
 	uuid "github.com/ukama/ukama/systems/common/uuid"
 )
  
  type Subscriber struct {
 	 SubscriberId          uuid.UUID `gorm:"primaryKey;type:uuid"`
 	 Name             string    `gorm:"size:255"`
 	 NetworkId             uuid.UUID `gorm:"type:uuid;index"`
-	 Email                 string    `gorm:"size:255;unique;not null;index"`
+	 Email                 string    `gorm:"size:255;not null;index"` 
```

Comment:
The unique index constraint combo the deletedAt should be added here. Not in ` cmd/server/main.go`

Reply:
Addressed.